src/app/_components/Badge.tsx

```diff
@@ -1,13 +1,14 @@
 import clsx from 'clsx'
 
 type BadgeProps = {
-  variant?: 'primary' | 'ghost'
+  variant?: 'primary' | 'ghost' | 'ghostBlue'
```

Comment:
I prefer if we don't use colors in variant names - This allows us to change theming in the future more easily. Normally, it would be nice to have something like `ghostInfo` - though for now we can go with `ghostBrand` - What do you think?

Reply:
Come to think of it - I don't we're using the`primary` style anymore.

We could go with `primary` (`ghost`) and `secondary` (`ghostBlue`) - Or whatever you think would be better?

Reply:
`primary` and `secondary` sound good  👍